src/routes/boot.ts

```diff
@@ -210,7 +212,7 @@ const loggedInBoot = async (
     flags: adjustFlagsToUser(flags, user),
     alerts: {
       ...excludeProperties(alerts, ['userId']),
-      changelog: alerts.lastChangelog < new Date(lastChangelog),
+      changelog: alerts.lastChangelog < new Date(lastChangelogMs),
```

Comment:
From what I can see locally, value returned from `getRedisObject` is string representation of `μs` timestamp that was stored inside.

`new Date` was returning `InvalidDate` and `changelog` was always `false`.

Reply:
I don't think Redis has a Date object even. Maybe it's something in the day we store the value.
We can save the date as an iso string to make sure we always read it the right way. Wdyt?

Reply:
It actually stores ISO strings with `setRedisObject(REDIS_CHANGELOG_KEY, post.createdAt)` but not sure where it gets converted to timestamp string, kinda weird.

Reply:
I think I found it, inside the worker `post` is of type `ChangeObject`, which changes `Date` to `number`, so this is where the sting timestamp comes from.